dist/build_deploy.sh

```diff
@@ -12,7 +12,7 @@ ensure_build_container "${DOCKERFILE_BUILD}"
 
 DOCKERFILE_DEPLOY="$ABSOLUTE_PATH/Dockerfile.deploy/Dockerfile"
 
-docker build -t "${IMAGE}:${IMAGE_TAG}" $DOCKERFILE_DEPLOY
+docker build -t "${IMAGE}:${IMAGE_TAG}" -f $DOCKERFILE_DEPLOY .
```

Comment:
Looks like we dropped `-f` in 071c112991c8 (#24).  What changed to make it required again?

Reply:
Ahh, de5b199531f (#222) shifted this from a `$RELEASE_OUTPUT_DIR` directory to a `$DOCKERFILE_DEPLOY` Dockerfile.

/lgtm

Reply:
In #24 the script was reworked to create a temp dir and copying binaries produced by `cargo build --release` there. In multistage build its no longer required, so the context is set to the whole dir and it requires the path to dockerfile